templates/myaccount/subscription-totals-table.php

```diff
@@ -4,7 +4,7 @@
  *
  * @package WooCommerce_Subscription/Templates
  * @since 1.0.0 - Migrated from WooCommerce Subscriptions v2.6.0
- * @version 1.0.0 - Migrated from WooCommerce Subscriptions v2.6.0
+ * @version 7.9.0 - Safety checks added to guard against fatal errors in certain unusual conditions
```

Comment:
```suggestion
 * @version 7.1.0
```

Just checking, this should be the version of WooCommerce Subscriptions these changes are shipped in (i.e. next version is 7.1). Is that correct?

Reply:
Yes, sorry I hadn't pieced that together at the time I made this change. Thanks!

Reply:
In fact, though I applied your change, this should now be `7.2.0`, right (because 7.1.0 just shipped)? I'll make that adjustment, but shout out if I'm mistaken